app/Controllers/ReservationsAdminController.php

```
@@ -194,6 +194,11 @@ public function createForm(Request $request, Response $response, mysqli $db): Re
             }
         }
 
+        $defaultLoanDays = (int) ((new \App\Models\SettingsRepository($db))->get('loans', 'loan_duration_days', '30') ?? 30);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '197', column '40'). (undefined)

(MissingImport)